Makefile.rules_generic

```diff
@@ -159,3 +160,6 @@ SCAN_BUILD ?= scan-build
 SCAN_BUILD_OUTPUT ?= $(CURDIR)/output-scan-build
 scan-build:
 	$(SCAN_BUILD) --use-cc=$(CC) -analyze-headers -enable-checker security -enable-checker unix -enable-checker valist -o $(SCAN_BUILD_OUTPUT) $(MAKE) default
+
+app.toml: bin/app.elf
```

Comment:
I'm not sure about the app.toml location, could it be somewhere else ? in bin ? in build/<target> ?

Reply:
I can try to modify it to have it in bin or build

Reply:
Done